### PR TITLE
slow down exp backoff from 1 ms to 100 ms factor

### PR DESCRIPTION
**Pull request by kalefranz**

### CODE REVIEW

**Comment by mcg1969** [comment]
Gotta love the single character commits

File: conda/install.py [hunk 1]
```diff
@@ -241,7 +241,7 @@ def exp_backoff_fn(fn, *args):
             if e.errno in (errno.EPERM, errno.EACCES):
                 if n == max_retries-1:
                     raise
-                time.sleep(((2 ** n) + random.random()) * 1e-3)
+                time.sleep(((2 ** n) + random.random()) * 1e-1)
```
Comment (mcg1969) [inline conda/install.py]: Cygwin's approach suggests we could do `time.sleep(0.025)` and `max_retries=40`. That would eliminate the dependency on `random`.

Comment (mingwandroid) [inline conda/install.py]: I'd like to replace the call to random() with ctypes rand() instead, that'll be very fast and have no dependencies.

Comment (kalefranz) [inline conda/install.py]: @mingwandroid Want to take a crack at another patch?

Comment (kalefranz) [inline conda/install.py]: I'm going to merge this as is to 4.1.x as an improvement.  But we can also improve from there.

**Comment by mingwandroid** [comment]
Hey @kalefranz, I think what we have is ok for now, though we may want to play it even safer and go back to seconds for bigger packages on slower computers.

Once I get qt5 builds out of the way I'll be happy to revise this, but I don't want to take my eye off that.

File: conda/install.py (hunk 1 above)
Comment (mcg1969) [inline conda/install.py]: I don't think we're going to be hitting the kernel that much. I'd love to see some statistics showing the average required backoff time and count for typical scenarios.

And do we really need rand at all? I suppose if it is already linked in it doesn't matter, and we already use ctypes. But I will bet a fixed backoff starting interval will suffice. Or maybe even a very simple pseudorandom source like the PID modulo 11.

**Comment by mcg1969** [comment]
How much backoff do we really need here? How slow are the virus scanners these days? Cygwin seems to be fine with 1 second total; we are at 1.5.

**Comment by mcg1969** [comment]
And remember, the scan starts at the beginning of the extraction; or clock starts at the end.

**Comment by jakirkham** [comment]
So, this should fix our Windows move issues correct?

**Comment by kalefranz** [comment]
Should, yes.

**Comment by mcg1969** [comment]
Well, I'm optimistic but not certain. Does AppVeyor use a virus scanner? I guess they probably do.

**Pull request merged**